src/scripts/settings/userSettingsBuilder.js

```diff
@@ -202,6 +202,20 @@ define(['appSettings', 'events'], function (appSettings, events) {
         return this.get('screensaver', false);
     };
 
+    UserSettings.prototype.libraryPageSize = function (val) {
+        if (val != null) {
```

Comment:
what if `val` is `undefined`?

Reply:
All of the options act like this. It's not optimal but it's outside of the scope of this PR, imo.